cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -554,9 +552,9 @@ public void testMetadata() throws Exception {
     testCommandOutputContains(cli, String.format("get metadata-tags %s scope system", fakeWorkflowId),
                               FakeWorkflow.FakeAction.ANOTHER_FAKE_NAME);
     testCommandOutputContains(cli, String.format("get metadata-tags %s scope system", fakeDsId),
-                              BatchReadable.class.getSimpleName());
+                              "batch");
```

Comment:
I think it is okay to use string in test case, since "batch" can be used for datasets or programs.